### PR TITLE
HOLD: Getting ready for the announcement

### PR DESCRIPTION
**Pull request by brandonhundt**
⏰ : DO NOT MERGE UNTIL MONDAY OCT 7 AT 1PM EST

- press release
- landing page
- banner
- some CTA work

Run app:

- [x] `$ bundle install`
- [x] `$ bundle exec jekyll server`
- [x] Go to http://localhost:4000/ and click the banner to end up at new banner to go to the pilot page
- [x] The Press Release is there, but Jekyll won't create the page until Monday because its dated as Monday (which will be fine when we merge it on Monday)

### CODE REVIEW

File: _posts/2023-07-06-dovetail-multiple-feeds.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ image: /assets/img/og-image.jpg
 heroimage: /img/image-multiple-feeds.jpg
 heroimagealt: Podcasting studio with microphone in foreground
 excerpt: Simplify podcast distribution and create new avenues for growth with PRX’s podcast publishing and monetization platform, Dovetail.
+cta: cta-demo.html
```
Comment (brandonhundt) [inline _posts/2023-07-06-dovetail-multiple-feeds.md:10]: The end CTA was hard coded into posts, so I cleaned this up, and now you can choose which one you want via the Front Matter.

Comment (cavis) [reply _posts/2023-07-06-dovetail-multiple-feeds.md:10]: Neat!

File: _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+permalink: /2024/10/07/prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacities
+canonical: /2024/10/07/prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacities
```
Comment (cavis) [inline _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:4]: Had to rename this file / backdate these - but did check this post as well. Looks good to me!

File: _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+title: PRX launches a pilot for public media stations to strengthen on-demand audio and podcasting capacities
+image: /assets/img/og-pilot.jpg
+heroimage: img/hero-pilot.jpg
+heroimagealt: Podcast studio with microphone in foreground
+excerpt: PRX Digital Audio Pilot for Public Media invites 30 stations nationwide to strengthen their infrastructure for on-demand audio and podcasting
+cta: cta-pilot.html
+
+---
+
+*Boston, MA (October 7, 2024)*–– [The Corporation for Public Broadcasting](https://cpb.org/) (CPB) awarded a $359,600 grant to public media distributor [PRX](http://www.prx.org) for a pilot that will enable free access to Dovetail for 30 public media stations. Dovetail is PRX’s digital platform that currently supports hundreds of podcasts, including more than 150 from stations nationwide.. Public media stations that participate in the pilot will test Dovetail’s impact on audience engagement and station revenue.
```
Comment (cavis) [inline _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:14]: "stations nationwide.." has 2 dots.

File: _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+heroimage: img/hero-pilot.jpg
+heroimagealt: Podcast studio with microphone in foreground
+excerpt: PRX Digital Audio Pilot for Public Media invites 30 stations nationwide to strengthen their infrastructure for on-demand audio and podcasting
+cta: cta-pilot.html
+
+---
+
+*Boston, MA (October 7, 2024)*–– [The Corporation for Public Broadcasting](https://cpb.org/) (CPB) awarded a $359,600 grant to public media distributor [PRX](http://www.prx.org) for a pilot that will enable free access to Dovetail for 30 public media stations. Dovetail is PRX’s digital platform that currently supports hundreds of podcasts, including more than 150 from stations nationwide.. Public media stations that participate in the pilot will test Dovetail’s impact on audience engagement and station revenue. 
+
+[Dovetail from PRX](/) enables public media stations and content producers to deliver audio content to audiences across all on-demand podcast platforms, provide real-time audience data and metrics, manage dynamic ad delivery, identify underwriting and fundraising opportunities, and integrate easily with existing technology.
```
Comment (cavis) [inline _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:16]: "... enables public media stations" ... to "deliver", "provide", "manage".

"provide" is a weird word in here - maybe "view" or something?

File: _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+[Dovetail from PRX](/) enables public media stations and content producers to deliver audio content to audiences across all on-demand podcast platforms, provide real-time audience data and metrics, manage dynamic ad delivery, identify underwriting and fundraising opportunities, and integrate easily with existing technology. 
+
+“CPB is committed to investing in innovative technologies that allow us to increase our reach and serve the American people with the trusted content important to their lives now and for the future,” said Patricia Harrison, president and CEO of CPB.
+
+“Public media needs its own shared platforms and services to ensure universal and equitable access, efficient operation, innovation, and collaboration,” said Kerri Hoffman, CEO of PRX. “Dovetail from PRX is infrastructure that can connect these dots. We’re proud to work in partnership with remarkable stations across the country and are grateful to CPB for this support.”
```
Comment (cavis) [inline _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:20]: I might say "is _the_ infrastructure".

Comment (cavis) [reply _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:20]: Ah, and in `pilot.md` it does say that.

File: pages/pilot.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      <li>Tools for rapidly identifying and forecasting underwriting, sponsorship, and monetization opportunities</li>
+      <li>Inventory management for ads, custom messages or alerts, digital fundraising drives, and promotions</li> 
+      <li>A values-aligned, privacy-first suite of applications</li>
+      <li>Integration with any existing technology infrastructure</li>
+    </ul>
+    </div>  
+</section>
+<section class="section bg-gray-x p-5">
+  <div class="container col-xxl-8">
+    <p>Stations of all sizes are eligible and space in the pilot is limited. CPB’s support means there would be no cost to your station for implementing Dovetail during this pilot period.</p>
```
Comment (cavis) [inline pages/pilot.md:44]: "implementing Dovetail" is odd to me.  Using?  Adopting?  Hrm... maybe it's just my developer brain assigning different meaning to that.

**Review by cavis — APPROVED** [review]
Links/pages all look good to me.  I gave you some bonus word smithing you probably don't need.  This looks good to me!

File: _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md (hunk 4 above)
Comment (brandonhundt) [reply _posts/2024-10-07-prx-launches-pilot-public-media-stations-strengthen-on-demand-audio-podcasting-capacitiesy-for-public-media.md:16]: This is agreed language between multiple parties ;)

**Pull request merged**